src/controllers/favorites.js

```diff
@@ -117,59 +128,82 @@ define(["appRouter", "cardBuilder", "dom", "globalize", "connectionManager", "ap
-                    Fields: "PrimaryImageAspectRatio,BasicSyncInfo",
-                    CollapseBoxSetItems: !1,
-                    ExcludeLocationTypes: "Virtual",
-                    EnableTotalRecordCount: !1
-                };
+        return function () {
+            var apiClient = this.apiClient;
+            var options = {
+                SortBy: (section.types, "SeriesName,SortName"),
+                SortOrder: "Ascending",
+                Filters: "IsFavorite",
+                Recursive: true,
+                Fields: "PrimaryImageAspectRatio,BasicSyncInfo",
+                CollapseBoxSetItems: false,
+                ExcludeLocationTypes: "Virtual",
+                EnableTotalRecordCount: false
+            };
             options.Limit = 20;
             var userId = apiClient.getCurrentUserId();
-            return "MusicArtist" === section.types ? apiClient.getArtists(userId, options) : (options.IncludeItemTypes = section.types, apiClient.getItems(userId, options))
-        }
+
+            if ("MusicArtist" === section.types) {
+                return apiClient.getArtists(userId, options);
+            }
+
+            if ("Person" === section.types) {
+                return apiClient.getPeople(userId, options);
+            }
+
+            options.IncludeItemTypes = section.types;
+            return apiClient.getItems(userId, options);
+        };
     }
 
     function getRouteUrl(section, serverId) {
         return appRouter.getRouteUrl("list", {
             serverId: serverId,
             itemTypes: section.types,
-            isFavorite: !0
-        })
+            isFavorite: true
+        });
     }
 
     function getItemsHtmlFn(section) {
-        return function(items) {
-            var supportsImageAnalysis = appHost.supports("imageanalysis"),
-                cardLayout = (appHost.preferVisualCards || supportsImageAnalysis) && section.autoCardLayout && section.showTitle;
-            cardLayout = !1;
-            var serverId = this.apiClient.serverId(),
-                leadingButtons = layoutManager.tv ? [{
-                    name: globalize.translate("All"),
-                    id: "more",
-                    icon: "&#xE87D;",
-                    routeUrl: getRouteUrl(section, serverId)
-                }] : null,
-                lines = 0;
-            return section.showTitle && lines++, section.showYear && lines++, section.showParentTitle && lines++, cardBuilder.getCardsHtml({
+        return function (items) {
+            var supportsImageAnalysis = appHost.supports("imageanalysis");
+            var cardLayout = (appHost.preferVisualCards || supportsImageAnalysis) && section.autoCardLayout && section.showTitle;
+            cardLayout = false;
+            var serverId = this.apiClient.serverId();
+            var leadingButtons = layoutManager.tv ? [{
+                name: globalize.translate("All"),
+                id: "more",
+                icon: "&#xE87D;",
```

Comment:
We should replace these with ASCII text, the unicode values are a pain to deal with. I haven't found an easy way to update all of them automatically though.

Reply:
PR: Replace all Unicode icon with there value https://github.com/jellyfin/jellyfin-web/pull/582